RRS/RRS/urls.py

```diff
@@ -28,4 +28,5 @@
     url(r'^', include('customer.urls', namespace='customer')),
     url(r'^staff/', include('staff.urls', namespace='staff')),
     url(r'^admin/', include(admin.site.urls)),
+    url(r'^test-dedicated/', include('test_dedicated.urls', namespace='test_dedicated')),
```

Comment:
In the future, could we commit this to a separate branch since it is different from the main bug issue?